content/chainguard/libraries/java/global-configuration.md

```diff
@@ -137,6 +137,101 @@ configuration](/chainguard/libraries/java/build-configuration). and build a firs
+
+1. Navigate to the **Secret Manager**
+1. Press **Create secret**. 
+1. Set the **Name** to *chainguard-libraries-java*.
+1. Use the **Password** from chainctl output to set the **Secret value**.
+1. Press **Create secret**.
+
+Navigate to Artifact Registry and select **Repositories** in the left hand
+navigation under the **Artifact Registry** label to configure a remote
+repository for the Maven Central Repository:
+
+1. Press **Create a Repository** or the **+** button.
+1. Set the **Name** to *central*.
+1. Set the **Format** to *Maven*.
+1. Select *Remote* for the **Mode**.
+1. Select *Maven Central* for the **Remote repository source**.
+1. Choose a suitable **Region** for your development in **Location type**.
+1. Press **Create**.
+
+Configure a remote repository for the Chainguard Libraries for Java repository:
+
+1. Press the **+** button to add another repository.
+1. Set the **Name** to *chainguard*.
+1. Set the **Format** to *Maven*.
+1. Select *Remote* for the **Mode**.
+1. Select *Custom* for the **Remote repository source**.
+1. Set the URL for the Custom repository to *https://libraries.cgr.dev/maven*.
+1. Select *Authenticated* in **Remote repository authentication mode**.
+1. Set **Username for the upstream repository** to the [value as retrieved
+   with chainctl](/chainguard/libraries/access/).
+1. Select the *chainguard-libraries-java* secret in the list for the **Secret** input.
+1. Choose the same suitable **Region** for your development in **Location type**
+   as configured for the *central* repository.
+1. Press **Create**.
+
+Combine the two repositories in a new virtual repository:
+
+1. Press the **+** button to add another repository.
+1. Set the **Name** to *chainguard-maven*.
+1. Set the **Format** to *Maven*.
+1. Select *Virtual* for the **Mode**.
+1. Press **Add upstream repository** in **Virtual upstream repositories**.
+1. Use the **Browse** button to locate and select the *chainguard* repository as
+   **Repository 1** and set the **Policy name 1** to *chainguard*.
+1. Use the **Browse** button to locate and select the *central* repository as
+   **Repository 1** and set the **Policy name 1** to *central*.
+1. Press **Add upstream repository** in **Virtual upstream repositories**.
+1. Set the **Priority** value for the *chainguard* policy name to a higher value
+   than the *central* priority value.
+1. Choose the same suitable **Region** for your development in **Location type**
+   as configured for the *central* repository.
+1. Press **Create**.
+
+### Build tool access
+
+The following steps allow you to configure your build tool for accessing the
+repository:
+
+1. Navigate to Artifact Registry and select **Repositories** in the left hand
+   navigation under the **Artifact Registry** label.
+1. Click on the *chainguard-maven* repository name in the list of repositories.
+1. Press the **Setup instructions** button and follow the documentation. Note
```

Comment:
This is quite elaborate and a pain, but also well documented there so we really should not replicate that documentation.